src/cbc_sdk/platform/alerts.py

```diff
@@ -84,47 +84,48 @@ class Alert(PlatformModel):
     }
 
     REMAPPED_ALERTS_V7_TO_V6 = {
```

Comment:
Two mappings are needed for process_guid

* Exists on all alert types in v7
* Must be mapped to process_guid on WATCHLIST type alerts in v6
* must also be mapped to threat_cause_process_guid when the alert type is CB_ANALYTIC